### ISSUE TITLE
Dismiss button <a> tag is not properly formatted for WCAG compatibility

### ISSUE THREAD

**Issue by Drikus** [osano/cookieconsent#693]
In the current version the 'got it' button is not accessible for keyboard users. It is able to get focus, but the keyboard user is not able to action the button.

Add href="#" to the link

To be compliant with accessability laws users need to be able to use all elements on a page by keyboard too.

**Comment by Drikus** [comment]
by the way, i've solved it now by adding the href attribute after the dismiss button is rendered using jquery as it's used in the project anyway.
`$('a.cc-btn').attr('href', '#');`
